packages/legend-graph/src/models/protocols/pure/v1/engine/execution/V1_ExecuteMappingTestInput.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { V1_PureModelContextData } from '../../model/context/V1_PureModelContextData';
+import { V1_pureModelContextDataPropSchema } from '../../transformation/pureProtocol/V1_PureProtocolSerialization';
+import type { V1_RawExecutionContext } from '../../model/rawValueSpecification/V1_RawExecutionContext';
+import { SerializationFactory, usingModelSchema } from '@finos/legend-shared';
+import { createModelSchema, optional, primitive } from 'serializr';
+import { V1_rawBaseExecutionContextModelSchema } from '../../transformation/pureProtocol/serializationHelpers/V1_RawValueSpecificationSerializationHelper';
+
+export class V1_ExecuteMappingTestInput {
+  clientVersion!: string;
+  mapping!: string;
+  testId!: string;
```

Comment:
I thought we said `testID` would be optional. If optional then run all the tests?

Reply:
we said we want testId to be a list , but if we want to run all then we send nothing in the list, but what we can do instead is just send 'all' in the testId and we can provide mechanism in the backend to run all tests.
Currently didnt implement that, there is another thing of what response to send in this case and how studio will display it

Reply:
I think we should do the list. If the lsit is empty or null run the full test suit. We should return the results as a list per each test run.